socorro/cron/jobs/ftpscraper.py

```diff
@@ -537,6 +538,12 @@ class FTPScraperCronAppDryRunner(App):  # pragma: no cover
         $ python socorro/cron/jobs/ftpscraper.py --product=mobile,thunderbird
 
     """
+    config_module = 'socorro.cron.jobs.ftpscraper.FTPScraperCronAppDryRunner.application_defaults'
+
+    class application_defaults:
+        database = DotDict({
+            'database_class': mock.MagicMock(),
+        })
```

Comment:
I don't think I ever use this class. There was a point where I was using it to debug some ftpscraper things, but then I discovered that ftpscraper calls a stored procedure and then a bunch of other things happen none of which happen in the dry-run effectively making dry-run unhelpful.

This class was much more important when there wasn't a local dev environment and you were running against -stage. But we don't do that anymore.

Anyhow, we can nix this. If you wanted to do that in this PR, that's cool. Otherwise I can write up a bug for it.